w3-admin.md

```diff
@@ -148,3 +150,44 @@ export const get = capability({
   },
 })
 ```
+
+### `trace/upload/get`
```

Comment:
I have suggested `trace/upload/add` so it corresponds to `upload/add`, meaning it's a capability to "trace upload/add" operation. Although I realize now it goes against established add, get, list, remove pattern used in other capabilities.

However `trace/upload/get` also feels counter intuitive to me even though I get the logic. On one hand it would make sense to have `upload/trace` and `store/trace`,  but that goes against prefix defining the ownership.

How about `console/upload/inspect` and `console/store/inspect` instead ? Alternatively it could be `admin/upload/inspect` or `system/upload/inspect`. I suppose `trace/upload/inspect` works too.

Reply:
chatted offline and agreed to go with `admin/{upload|store}/inspect`